app/controllers/books_controller.rb

```diff
@@ -17,6 +17,9 @@ def edit
 
   def create
     @book = Book.new(book_params)
+    create_publisher
+    create_authors
+    create_categories
 
     respond_to do |format|
```

Comment:
did you know that you can remove respond_to and format.html and the code will still works as intended?
remove it or keep it. Your call